commands/suggest_builders.go

```diff
@@ -0,0 +1,22 @@
+package commands
+
+import (
+	"github.com/buildpack/pack/logging"
+	"github.com/spf13/cobra"
+)
+
+func SuggestBuilders(logger *logging.Logger) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "suggest-builders",
+		Short: "Display list of builders recommended by the Cloud Native Buildpacks project",
```

Comment:
Maybe shorten to: `Display list of recommended builders`?

Reply:
I put exactly what the issue acceptance criteria states. Maybe @mgibson1121 can weigh in on this. I'm good to change it to whatever makes the most sense.

Reply:
Ah, my bad. Didn't realize that. I'll leave it to @mgibson1121 then :)

Reply:
I'm fine with @ameyer-pivotal's suggestion